### DISCUSSION TITLE
Was anyone able to run successfully Podman baked w/ the last build in the UDM PRO SE?

### DISCUSSION THREAD

**Discussion by mazzy89** [unifi-utilities/unifi-common#274] [Q&A] [answered]
I have found out that the latest Podman baked w/ the build in this repo does not work on UDM PRO SE. I have found endless issues.

First the storage. I had to set `vfs`. then the `containers.conf` has a typo for the `cgroups` parameter but this was easy to spot. Now I can't run any containers.  I just get

```
root@UDM-PRO-SE:/etc/containers# podman run docker.io/library/httpd
ERRO[0000] cannot start a container that has stopped
Error: `/usr/bin/runc start 40b6c0b453a7f8c2d0d8dbfe038a7d9a89ca62bc85f5128c2d3608e54f3bd3c3` failed: exit status 1
```

**Comment by mazzy89** [comment, marked as answer]
this has been solved. take a look here Cgroup configuration typo for Podman on UDM Pro SE

**Comment by dibun75** [comment]
Is there any easy howto guide as I cannot find any

**Reply by mazzy89** [reply]
Just download the zip file produced by the build and unpack it in te root of your udm pro se. This is enough to have podman running

**Reply by automatestuff** [reply]
Being very new to this ... can you please provide instructions?

**Reply by mazzy89** [reply]
https://github.com/boostchicken-dev/udm-utilities/commit/c1b8b5ec71bf42329c502849cc20bd884c11af6a

**Reply by janthony6** [reply]
having a hard time with this too. the guide makes it look like podman has to be manually installed and I have no idea how to download it or do it. or do I just run this and podman will be installed?

```bash
 curl -fsL "https://raw.githubusercontent.com/boostchicken/udm-utilities/HEAD/on-boot-script/remote_install.sh" | /bin/sh
 ```

**Reply by mazzy89** [reply]
You can download it as the artifact of the last github action build of this project. Then decompress the zip in the root and it will be installed.

**Reply by janthony6** [reply]
okay. so this is the link: 
https://github.com/boostchicken-dev/udm-utilities/suites/4551796675/artifacts/122216505 

what do I enter in the udmse to download from that link? and then to extract it?

**Reply by mazzy89** [reply]
Here the link from the README https://github.com/boostchicken/udm-utilities/actions download the artifacts unzip in the root of your udm pro se device and thats it

**Reply by emilvissing** [reply]
@mazzy89 but not all files necessary to have podman working are actually included in that zip, right....? storage.conf, policy.json etc.

**Reply by amomchilov** [reply]
@emilvissing Same issue here, IDK where to get `policy.json` from. Cross-reference: https://github.com/boostchicken-dev/udm-utilities/issues/321#issuecomment-1093525964

**Comment by jj358mhz** [comment]
When I unzip `udmse-podman-install.zip.zip`, I now see another zip called `podman-install.zip` in the root. Do I leave `podman-install.zip` as is, or do I need to unzip it as well?

![image](https://user-images.githubusercontent.com/6722828/152040893-88150148-ba87-49d6-8b11-0ae0556569fa.png)

**Reply by mazzy89** [reply]
Yes it's called matrioska 😂😂😂

Joking aside this is because John in the github actions of the podman relaase created as an artifact of the release a zip. So zipping a zip, you get of course as zip.

You need to unzip it as well

**Comment by StreborStrebor** [comment]
Just to be sure, If I only want to have on_boot on my UDM-SE (to run a few scripts that change MTU values), then I don't need podman, right? Or does on-boot require podman?

My aim is to keep the config as simple as possible on the UDM-SE, without unnecessary installs/configs.

**Reply by mazzy89** [reply]
on-boot is nothing than a systemd unit service that run all the `sh` scripts present on a location. that's it.
Podman is required in all those cases that a container is needed to be run from many different use cases like adguard, pihole, etc...

Now for your case like changing MTU values, unless you are doing that using a container, you won't need podman.

**Reply by StreborStrebor** [reply]
Thank you for the amazingly fast answer @mazzy89

**Reply by guysbrown** [reply]
I take it the case is the same if I only want to install the fan speed script (https://github.com/renedis/ubnt-auto-fan-speed).

No need for Podman, just install the boot script and the fan speed script and all it good?
